src/main/java/flaxbeard/steamcraft/integration/BotaniaIntegration.java

```diff
@@ -129,6 +129,8 @@ public static Multimap addModifiers(Multimap map, ItemStack stack, int armorType
                     break;
                 case 4:
                     hp = 2;
+                default: 
+                	break;
```

Comment:
If you look in the file via text editor here, it looks like how  is seen in my IDE. I this  web page appearance is misleading.